### PR TITLE
Common changes documentation

### PR DESCRIPTION
**Pull request by shanmydell**
Common changes documentation

### CODE REVIEW

**Review by rajendraindukuri — APPROVED** [review]

File: content/docs/csidriver/installation/helm/powerflex.md [hunk 1]
```diff
@@ -104,7 +104,7 @@ kubectl -n kube-system kustomize deploy/kubernetes/snapshot-controller | kubectl
 ```
 
 *NOTE:*
-- When using Kubernetes 1.21/1.22/1.23 it is recommended to use 5.0.x version of snapshotter/snapshot-controller.
+- When using Kubernetes 1.21/1.22/1.23 it is recommended to use 6.0.x version of snapshotter/snapshot-controller.
```
Comment (bharathsreekanth) [inline content/docs/csidriver/installation/helm/powerflex.md:107]: should this be 1.22, 23 and 24 since that is our supported platforms?

**Review by bharathsreekanth — CHANGES_REQUESTED** [review]

**Review by panigs7 — APPROVED** [review]

File: content/docs/csidriver/installation/helm/powermax.md [hunk 2]
```diff
@@ -185,7 +185,8 @@ CRDs should be configured during replication prepare stage with repctl as descri
 4. Create the secret by running `kubectl create -f samples/secret/secret.yaml`.
 5. If you are going to install the new CSI PowerMax ReverseProxy service, create a TLS secret with the name - _csireverseproxy-tls-secret_ which holds an SSL certificate and the corresponding private key in the namespace where you are installing the driver.
 6. Copy the default values.yaml file `cd helm && cp csi-powermax/values.yaml my-powermax-settings.yaml`
-7. Edit the newly created file and provide values for the following parameters `vi my-powermax-settings.yaml`
+7. Ensure the unisphere have 10.0 REST endpoint support.
```
Comment (AronAtDell) [inline content/docs/csidriver/installation/helm/powermax.md:188]: How do they ensure this?

File: content/docs/csidriver/installation/operator/powermax.md [hunk 3]
```diff
@@ -36,6 +36,35 @@ Set up the iSCSI initiators as follows:
...
+no_path_retry 10
+```
+
+#### PowerPath for Linux requirements
+
+CSI Driver for Dell PowerMax supports PowerPath for Linux. Configure Linux PowerPath before installing the CSI Driver.
+
+Set up the PowerPath for Linux as follows:
+
+- All the nodes must have the PowerPath package installed . Download the PowerPath archive for the environment from [Dell EMC Online Support](https://www.dell.com/support/home/en-in/product-support/product/powerpath-for-linux/drivers).
```
Comment (AronAtDell) [inline content/docs/csidriver/installation/operator/powermax.md:65]: Remove all instance of EMC. Thanks!

File: content/docs/snapshots/volume-group-snapshots/_index.md [hunk 4]
```diff
@@ -6,6 +6,8 @@ Description: >
   Volume Group Snapshot module of Dell CSI drivers
 ---
 ## Volume Group Snapshot Feature
+The Dell CSM Volume Group Snapshotter is an operator which extends Kubernetes API to support crash-consistent snapshots of groups of volumes.
+Volume Group Snapshot supports Powerflex and Powerstore driver.
```
Comment (AronAtDell) [inline content/docs/snapshots/volume-group-snapshots/_index.md:10]: Should be PowerStore.

File: content/docs/csidriver/features/powerscale.md [hunk 5]
```diff
@@ -22,7 +22,7 @@ You can use existent volumes from the PowerScale array as Persistent Volumes in
 1. Open your volume in One FS, and take a note of volume-id.
 2. Create PersistentVolume and use this volume-id as a volumeHandle in the manifest. Modify other parameters according to your needs.
 3. In the following example, the PowerScale cluster accessZone is assumed as 'System', storage class as 'isilon', cluster name as 'pscale-cluster' and volume's internal name as 'isilonvol'. The volume-handle should be in the format of <volume_name>=_=_=<export_id>=_=_=<zone>=_=_=<cluster_name>
-4. If Quotas are enabled in the driver, it is recommended to add the Quota ID to the description of the NFS export in the following format: 
+4. If Quotas are enabled in the driver, it is required to add the Quota ID to the description of the NFS export in the following format:
```
Comment (rsedlock1958) [inline content/docs/csidriver/features/powerscale.md:25]: Change "the following" to "this"

File: content/docs/csidriver/installation/operator/powermax.md [hunk 6]
```diff
@@ -36,6 +36,35 @@ Set up the iSCSI initiators as follows:
 
 For more information about configuring iSCSI, see [Dell Host Connectivity guide](https://www.delltechnologies.com/asset/zh-tw/products/storage/technical-support/docu5128.pdf).
 
+#### Linux multipathing requirements
+
+CSI Driver for Dell PowerMax supports Linux multipathing. Configure Linux multipathing before installing the CSI Driver.
+
+Set up Linux multipathing as follows:
```
Comment (rsedlock1958) [inline content/docs/csidriver/installation/operator/powermax.md:43]: Change this sentence to read "Follow this procedure to set up Linux multipathing:

File: content/docs/csidriver/installation/operator/powermax.md [hunk 7]
```diff
@@ -36,6 +36,35 @@ Set up the iSCSI initiators as follows:
...
+
+- All the nodes must have the _Device Mapper Multipathing_ package installed.  
+  *NOTE:* When this package is installed it creates a multipath configuration file which is located at `/etc/multipath.conf`. Please ensure that this file always exists.
+- Enable multipathing using `mpathconf --enable --with_multipathd y`
+- Enable `user_friendly_names` and `find_multipaths` in the `multipath.conf` file.
+
+As a best practice, use the following options to help the operating system and the mulitpathing software detect path changes efficiently:
```
Comment (rsedlock1958) [inline content/docs/csidriver/installation/operator/powermax.md:50]: Change "the following" to "these"

File: content/docs/csidriver/installation/operator/powermax.md [hunk 8]
```diff
@@ -36,6 +36,35 @@ Set up the iSCSI initiators as follows:
...
+features "1 queue_if_no_path"
+path_selector "round-robin 0"
+no_path_retry 10
+```
+
+#### PowerPath for Linux requirements
+
+CSI Driver for Dell PowerMax supports PowerPath for Linux. Configure Linux PowerPath before installing the CSI Driver.
+
+Set up the PowerPath for Linux as follows:
```
Comment (rsedlock1958) [inline content/docs/csidriver/installation/operator/powermax.md:63]: Change this sentence to read "Follow this procedure to set up PowerPath for Linux:

File: content/docs/csidriver/installation/test/unity.md [hunk 9]
```diff
@@ -28,9 +28,9 @@ You can find all the created resources in `test-unity` namespace.
    kubectl delete -f ./test/sample.yaml
    ```
 
-## Support for SLES 15 SP2
+## Support for SLES 15
 
-The CSI Driver for Dell Unity XT requires the following set of packages installed on all worker nodes that run on SLES 15 SP2.
+The CSI Driver for Dell Unity XT requires the following set of packages installed on all worker nodes that run on SLES 15.
```
Comment (rsedlock1958) [inline content/docs/csidriver/installation/test/unity.md:33]: Change "the following" to "these"

**Review by rsedlock1958 — COMMENTED** [review]
I have reviewed this. There are several minor edits.

File: content/docs/csidriver/installation/helm/powerflex.md (hunk 1 above)
Comment (shanmydell) [reply content/docs/csidriver/installation/helm/powerflex.md:107]: resolved

File: content/docs/csidriver/installation/operator/powermax.md (hunk 3 above)
Comment (shanmydell) [reply content/docs/csidriver/installation/operator/powermax.md:65]: updated

File: content/docs/snapshots/volume-group-snapshots/_index.md (hunk 4 above)
Comment (shanmydell) [reply content/docs/snapshots/volume-group-snapshots/_index.md:10]: Done

File: content/docs/csidriver/installation/test/unity.md (hunk 9 above)
Comment (shanmydell) [reply content/docs/csidriver/installation/test/unity.md:33]: done

File: content/docs/csidriver/installation/operator/powermax.md (hunk 7 above)
Comment (shanmydell) [reply content/docs/csidriver/installation/operator/powermax.md:50]: done

File: content/docs/csidriver/installation/operator/powermax.md (hunk 6 above)
Comment (shanmydell) [reply content/docs/csidriver/installation/operator/powermax.md:43]: done

File: content/docs/csidriver/features/powerscale.md (hunk 5 above)
Comment (shanmydell) [reply content/docs/csidriver/features/powerscale.md:25]: done

**Review by bharathsreekanth — APPROVED** [review]

File: content/docs/csidriver/installation/helm/powermax.md (hunk 2 above)
Comment (shanmydell) [reply content/docs/csidriver/installation/helm/powermax.md:188]: updated

**Review by boyamurthy — APPROVED** [review]

**Review by rajendraindukuri — APPROVED** [review]

**Pull request merged**